airflow/load_data_forecast.py

```diff
@@ -0,0 +1,102 @@
+from datetime import timedelta
+# The DAG object; we'll need this to instantiate a DAG
+from airflow import DAG
+# Operators; we need this to operate!
+from airflow.operators.bash_operator import BashOperator
+from airflow.operators.python_operator import PythonOperator
+from airflow.utils.dates import days_ago
+from airflow.models import Connection
+from airflow import DAG, settings
+# Connects to GoogleCloud
+from airflow.contrib.hooks.gcs_hook import GoogleCloudStorageHook
+# These args will get passed on to each operator
+import json 
+import pandas as pd
+import numpy as np
+# Needed for load_data
+from corona_ts.data_utils.data_crawler import load_data
+
+def add_gcp_connection(**kwargs):
+    new_conn = Connection(
+            conn_id="google_cloud_default",
+            conn_type='google_cloud_platform',
+    )
+    extra_field = {
+        "extra__google_cloud_platform__scope": "https://www.googleapis.com/auth/cloud-platform",
+        "extra__google_cloud_platform__project": "task-ts",
+        "extra__google_cloud_platform__key_path": '/home/efawe/airflow/dags/task-ts-562f84386e61.json'
```

Comment:
I'd probably prefer that this was set as environment variable in case we need to change it later i.e `os.environ["GCP_KEY_PATH"]`